src/scriptsys/scr_cvar.h

```diff
@@ -0,0 +1,8 @@
+#include "script.h"
+#include "common/macro.h"
+
+PIM_C_BEGIN
+
+void scr_cvar_init(lua_State* L);
```

Comment:
Would be good to have a matching shutdown for any init function, so that we can hot reload sections of the engine.